### DISCUSSION TITLE
Support for BeagleWire FPGA Cape (ICE40HX4K)

### DISCUSSION THREAD

**Discussion by ril3y** [FPGAwars/icestudio#836] [New Boards Requests]
Is there any plans on adding support for the BeagleWire FPGA board?
https://elinux.org/BeagleBoard/BeagleWire
If not is there any documentation on adding new board support?   I am not sure I have the skills needed but if there is some docs I will take a shot at it.

Thanks

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
